### PR TITLE
Feature/header container test

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: src/pages/Dashboard/__tests__/container.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      const { searchQuery } = instance.props;
+      const page = 2;
+      instance.checkSearchState(page);
+      expect(searchMovies).toHaveBeenCalledWith(searchQuery, page);
+    });
+
+    it('has empty search query', () => {
+      wrapper.setProps({ searchQuery: '' });
+      instance.checkSearchState();
+      expect(fetchTrendingMovies).toHaveBeenCalled();
```
Comment (plzen) [inline src/pages/Dashboard/__tests__/container.test.js]: Check values passed to function

File: src/pages/Home/__tests__/container.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    it('Should render dashboard', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
+  });
+
+  describe('User is not logged in.', () => {
+    it('Should render login', () => {
+      isUserLoggedIn.mockImplementation(() => false);
+      const wrapper = shallow(<ConnectedHome store={store} />).dive().dive();
```
Comment (plzen) [inline src/pages/Home/__tests__/container.test.js]: Move this line to describe block like you have in previous test, this is more convenient to have similar approach in all tests

File: src/pages/Dashboard/__tests__/container.test.js (hunk 1 above)
Comment (pink-bunny) [reply src/pages/Dashboard/__tests__/container.test.js]: Fixed.
Passes page to checkSearchState

File: src/pages/Home/__tests__/container.test.js (hunk 2 above)
Comment (pink-bunny) [reply src/pages/Home/__tests__/container.test.js]: Done

**Pull request merged**